src/Manifest.js

```diff
@@ -283,6 +292,13 @@ function Manifest(output, path) {
 Manifest.ANDROID_DEFAULT_PERMISSIONS = [ "ACCESS_NETWORK_STATE", "ACCESS_WIFI_STATE", "INTERNET" ];
 
 /**
+ * Default codepage to use is "western european".
+ * When using ISO 8859-1 unsupported characters in the manifest,
+ * the codepage needs to be adjusted accordingly.
+ */
+Manifest.WINDOWS_DEFAULT_CODEPAGE = "1252";
```

Comment:
@kenchris because Windows (WiX) does not support it for strings that are displayed in the installer UI.

You have "lang" so if ```"lang": "cn"``` you can convert UTF-16 to the codepage required for CN. That would be the proper fix